### PR TITLE
ref: update system config paths

### PR DESCRIPTION
**Pull request by jorgeorpinel**
per https://github.com/iterative/dvc.org/pull/2311#pullrequestreview-615705927

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
This should fix the formatting in `master` @shcheklein, please merge 🙂

**Comment by shcheklein** [comment]
8 files were broken?

File: content/docs/command-reference/cache/dir.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ cache directory.
 - `--global` - modify the global config file (e.g. `~/.config/dvc/config`)
   instead of the project's `.dvc/config`.
 
-- `--system` - modify the system config file (e.g. `/etc/xdg/dvc/config`) instead
-  of `.dvc/config`.
+- `--system` - modify the system config file (e.g. `/etc/xdg/dvc/config`)
+  instead of `.dvc/config`.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/cache/dir.md:36]: This one was broken by my formatting of #2311.

**Comment by shcheklein** [comment]
Hmm. should we revert that single file? I'm lost a bit in this PR, I guess :)

File: content/docs/command-reference/cache/dir.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/cache/dir.md:36]: The other 7 apply the same change to all the refs. as the title of this PR suggests. Sorry for the confusion, should I split this into 2 PRs?

**Comment by jorgeorpinel** [comment]
OK one sec, I'll split it... But I'm not trying to revert, I'm just fixing the formatting. The change in #2311 is good (and needs to get to all the other files after this).

**Comment by shcheklein** [comment]
So, it's not 8 files by mistake? Then let's merge it, I'm fine with it.

**Review by shcheklein — APPROVED** [review]

**Comment by jorgeorpinel** [comment]
Well I already removed the other 7 😢. The good news is that `master`'s formatting is good now. https://app.circleci.com/pipelines/github/iterative/dvc.org/8394/workflows/05c4ba3b-7844-4009-b6b4-362a6f106151

Anyway, the rest are already being changed in #2306 (which is a sort of stash for now — needs splitting).

**Comment by jorgeorpinel** [comment]
OK I submitted #2316 since I had a branch already, to get this done 🙂

**Pull request merged**